controller/src/beerocks/bml/internal/bml_internal.cpp

```diff
@@ -695,6 +695,24 @@ int bml_internal::process_cmdu_header(cmdu_vs_action_header_t beerocks_header,
                                 "is waiting...";
             }
         } break;
+        case beerocks_message::ACTION_BML_SET_CERTIFICATION_MODE_RESPONSE: {
+            //Signal any waiting threads
+            if (!wake_up(beerocks_message::ACTION_BML_SET_CERTIFICATION_MODE_REQUEST, 0)) {
+                LOG(WARNING) << "Received ACTION_BML_SET_CERTIFICATION_MODE_RESPONSE response, "
+                                " but no one is waiting...";
+            }
+        } break;
+        case beerocks_message::ACTION_BML_GET_CERTIFICATION_MODE_RESPONSE: {
+            auto response =
+                cmdu_rx.addClass<beerocks_message::cACTION_BML_GET_DFS_REENTRY_RESPONSE>();
```

Comment:
This doesn't seem right... Copy & paste error?

Reply:
yes. fixed.